Firebend.AutoCrud.Web.Sample/DbContexts/DbFunctionsExtensions.cs

```diff
@@ -0,0 +1,18 @@
+using System;
```

Comment:
if these are not supported natively in EF Core now, would it be worth while to move them [here](https://github.com/firebend/auto-crud/tree/main/Firebend.AutoCrud.EntityFramework/CustomCommands) so they can be utilized by a wider audience?

Reply:
yeah, sounds good

Reply:
Moved 52205f1